### DISCUSSION TITLE
SMN CN page comment field for variant instead of using full case comment field

### DISCUSSION THREAD

**Discussion by Jakob37** [Clinical-Genomics/scout#5291] [Ideas]
**Is your feature request related to a problem in the current program to new available techology or software? Please describe and add links/citations if appropriate.**

Our local geneticists are looking into using the SMN CN module in their clinical routine, and raised some points for discussion. I'll open issues for their discussion points to discuss.

**Describe the solution you'd like**

The comment field is for the case, while SNV-variants have their own specific case -> variant comment field.

I.e. here I tried adding a comment on the case, which also appears on the variant page. On the regular variant page, there is a specific comment box with an optional "Comment globally" checkbox.

How it looks at the SMN CN page:

![smn_cn](https://github.com/user-attachments/assets/acffcf14-223c-4be3-9f5e-ab35920cbdf0)

How it looks at the individual variant page:

![snv_comment](https://github.com/user-attachments/assets/6c0c562c-c26f-440d-898e-7bdb2d8d4462)

They asked for whether a case -> variant specific comment box could be included also in the case of SMN CN, and whether the global case related fields are removed.

**Comment by dnil** [comment]
Variant comments can be either local or global. Case comments are always only related to the case. The comments on the SMN page are case level comments.

There are no variants to be seen on the SMN page. See discussion on #4820.
